Cargo.toml

```diff
@@ -20,6 +20,7 @@ pyo3-build-config = { version = "0.24.1", optional = true }
 [dependencies]
 log = { version = "0.4.17", optional = true }
 pyo3 = { version = "0.24.1", features=["extension-module", "abi3-py37"], optional = true }
+chrono = { version = "0.4", optional = true }
```

Comment:
Hmm, I'm concerned about the versioning here. What happens when `chrono 0.5` is released, and we want to update this? It seems like changing the version here would require bumping the major version on `redb`.

I'm not familiar with how this is typically handled in the ecosystem. Can you find some examples?

Reply:
Honestly, I have zero idea. We could do chrono_v4 as the package name and support both until we are ready to do a major version bump, and at that point, we drop it.